ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6605,4 +6605,13 @@ VMADC:
   - mixed_air_damper_percentage_command
   - mixed_air_damper_percentage_sensor
   implements:
-  - CONTROL
\ No newline at end of file
+  - CONTROL
+
+CPM2X:
```

Comment:
I would expect these to have commands with them. The statuses are not very helpful on their own.

Reply:
I believe these pumps are cycled automatically, for example: pump A used for 1 day, B used the next day etc